tests/e2e_tests/common/user_steps.py

```diff
@@ -118,12 +132,16 @@ def create_service(self, service_name, protocol):
                 }
                 ]
                 }
+        if attributes:
+            service['services'][0]['attributes'] = attributes
+        if static_attributes:
+            service['services'][0]['static_attributes'] = static_attributes
         req = api.post_service(service, headers)
         assert req.status_code == 201, 'ERROR: ' + req.text + "El servicio {} no se ha creado correctamente".format(service_name)
         world.service_exists = True            
         return req
 
-    def create_service_with_params(self, service_name, service_path, resource, apikey, cbroker, entity_type={}, token={}):
+    def create_service_with_params(self, service_name, service_path, resource={}, apikey={}, cbroker={}, entity_type={}, token={}, attributes={}, static_attributes={}, protocol={}):

```

Comment:
Consider to hava a single "create_service_with_params" method, both for creating services with paramateres and without parameters